src/Components/Aspire.MongoDB.Driver/ConfigurationSchema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "definitions": {
+    "logLevel": {
+      "properties": {
+        "MongoDB": {
+          "$ref": "#/definitions/logLevelThreshold"
+        },
+        "MongoDB.Driver": {
+          "$ref": "#/definitions/logLevelThreshold"
+        },
+        "MongoDB.Driver.MongoClient": {
+          "$ref": "#/definitions/logLevelThreshold"
+        }
+      }
+    }
+  },
+  "properties": {
+    "Aspire": {
+      "type": "object",
+      "properties": {
+        "MongoDB": {
+          "type": "object",
+          "properties": {
+            "ConnectionString": {
```

Comment:
Is this the only setting worth exposing? There must be more 😄

Reply:
Hi @davidfowl ! 
I'll add more settings here.
Also, I've tried to add healthcheck support but the package `AspNetCore.HealthChecks.MongoDb` is not restored by the configured package sources. Is there any problem to use this package here?

Reply:
The package will need to be added to our internal mirror.

Reply:
I've implemented a custom health check for that but if you prefer I can change it to use the package when it available.

Reply:
Hi @eerhardt I've already tried to use this health check but the package is not available.

https://github.com/dotnet/aspire/pull/986#discussion_r1404285508
https://github.com/dotnet/aspire/pull/986#discussion_r1406403154

Reply:
Ah, sorry I missed that. The package needs to be mirrored to our trusted feed. I'm doing that now. It should be done in a few minutes.

Reply:
I've changed to use the package, please review.